### DISCUSSION TITLE
Upgrade to Preview 8, unable to run on Mac

### DISCUSSION THREAD

**Discussion by CaiNiaoM** [AvaloniaUI/Avalonia#11856] [Q&A]
I use Preview version 4, packaged as an app that can run on a Mac, but upgrading from Preview version 4 to Preview version 8 does not allow the program to run on a Mac system. Do I need to refer to anything else?

**To Reproduce**

**Expected behavior**
Please tell me how to upgrade to Preview 8 so that my application can run on a Mac

**Screenshots**
I only used these components
![1687341915376](https://github.com/AvaloniaUI/Avalonia/assets/52517354/22e21db5-08cc-49bd-97ff-5d18f88e48e1)

**Desktop (please complete the following information):**

- OS: [e.g. Mac]
- Version [e.g. 0.11.0-preview8 ]

**Additional context**

**Comment by hez2010** [comment]
Can you try rc1.1?

**Comment by timunie** [comment]
@CaiNiaoM I think this is more a question and not a bug. Thus I'll convert it into a Q&A of it. In general, please use Discussions or join us on telegram for how-to questions. Thank you.

**Comment by timunie** [comment]
Back to your topic. Do you get any new error message? In general we have many breaking changes between preview4 and 8 / RC1. Make sure to read through them:
- https://github.com/AvaloniaUI/Avalonia/wiki/Avalonia-11-Porting-Guide
- https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes#010--110

**Comment by Coloryr** [comment]
p6开始虚拟机跑不了
https://github.com/AvaloniaUI/Avalonia/discussions/10761

**Comment by hez2010** [comment]
If you are running macOS in a virtual machine then it is expected because we don't support running on macOS environment without a GPU.
